.github/renovate.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
We use renovate to keep dependencies updated but obviously I don't expect you to want this so let me know and I'll remove this